lib/addons/adot/otel-permissions.yaml

```diff
@@ -0,0 +1,155 @@
+apiVersion: v1
```

Comment:
do you see how we can maintain this file if it changes upstream? I am a bit concerned that it may be coupled to the specific helm version.
We also manually modified it from the original content.

Reply:
Also we talked about submitting an issue against the EKS add-on to apply RBAC automatically, so may be that could be our path to resolve this.

Reply:
I agree to what you are saying, I only removed two commented lines which had dashes to avoid rendering issue but long term maintenance is a concern but at this point we can have it as it is. I have submitted an issue with AWS OTEL Github which i will keep a watch, once they fix the Addon, i can also make a fix.
https://github.com/aws-observability/aws-otel-community/issues/79